dozer-api/protos/internal.proto

```diff
@@ -37,17 +37,33 @@ message ApplicationDetail {
   repeated SourceInfo sources = 5;
   repeated EndpointInfo endpoints = 6;
   string home_dir = 7;
+  Flags flags = 8;
 }
 
+message Flags {
+  bool dynamic = 1;
+  bool grpc_web = 2;
+  bool authenticate_server_reflection = 3;
+  bool push_events = 4;
```

Comment:
I think we have `push_events` `3` and `authenticate_server_reflection` `4`

Reply:
right !!, Update order in `dozer-types` also